ClientTests/RatingPromptManagerTests.swift

```diff
@@ -16,6 +15,7 @@ class RatingPromptManagerTests: XCTestCase {
     var promptManager: RatingPromptManager!
     var mockProfile: MockProfile!
     var createdGuids: [String] = []
+    var sentry: CrashingMockSentryClient!
```

Comment:
qq: is `CrashingMockSentryClient` a mock sentry client for testing? @yoanarios

Reply:
yes @nbhasin2, it's only used in RatingPromptTest for the logic to show rating prompt